src/operator/mshadow_op.h

```diff
@@ -415,11 +415,31 @@ MXNET_UNARY_MATH_OP(log_sigmoid, math::log(1.0f / (1.0f + math::exp(-a))));
 
 MXNET_UNARY_MATH_OP(log_sigmoid_grad, 1.0f / (1.0f + math::exp(a)));
 
-MXNET_UNARY_MATH_OP(mish, a * math::tanh(math::log(1.0f + math::exp(a))));
+struct mish : public mxnet_op::tunable {
+  template<typename DType>
+  MSHADOW_XINLINE static DType Map(DType a) {
+    // reference softrelu
+    auto softrelu = math::log1p(math::exp(a));
```

Comment:
And similarly here you should be able to use
```
auto softrelu = math::softrelu(a)
```
right? This way the logic stays in 1 place and is easier to maintain.

Reply:
Hi, @ptrendx . There is no `softrelu` in `src/operator/math_functions-inl.h`(namespace math).